src/main/java/frc/robot/LogitechGamingPad.java

```diff
@@ -286,12 +262,11 @@ public boolean checkDPad(int index)
 	 * @return If the angle is the same as the D-pad.
 	 */
 
-	public boolean checkDPad(double angle, boolean inDegrees)
-	{
-		if (!inDegrees)
-			angle = Math.toDegrees(angle);
-		return (int)angle == gamepad.getPOV(); 
-	}
+  public boolean checkDPad(double angle, boolean inDegrees) {
```

Comment:
I know this isn't your code that you've written, but imo `angle` should be in consistent units, and shouldn't change depending on the `inDegrees` argument. this makes the code that calls into `checkDPad` more readable